packer/add_IP_property_to_ova.sh

```diff
@@ -21,8 +23,15 @@ OVA=$1
 BASENAME=${OVA%.*}
 OVF=$"${BASENAME}.ovf"
 
-rm -f $"${BASENAME}.mf" # remove checksum file, otherwise, existing mf file will prevent ovftool coverting
-echo "covert ${OVA} to OVF file, named ${OVF}"
+### validate the OVA
+FILE_TYPE=`file ${OVA} | awk -F": " '{print $2}'`
+if "${FILE_TYPE}" != "POSIX tar archive"; then
+    echo "[Error]:  ${OVA} is not a valid OVA file"
+    exit -1
+fi
```

Comment:
Had an error in the example that had earlier.   This worked in my bash script, but please test out your code to make sure.

```
if [[ "${FILE_TYPE}" != *"tar archive"* ]]; then
    echo "[Error]:  ${OVA} is not a valid OVA file"
    exit -1
fi
```